package-lock.json

```diff
@@ -442,6 +444,292 @@
         "@theia/ext-scripts": "1.64.0"
       }
     },
+    "examples/api-samples/node_modules/@modelcontextprotocol/sdk": {
```

Comment:
This package-lock creates duplicates of already hoisted dependencies. Please make sure that this does not happen.